packages/web-components/src/components/va-segmented-progress-bar/va-segmented-progress-bar.tsx

```diff
@@ -63,7 +63,7 @@ export class VaSegmentedProgressBar {
           aria-valuemin="0"
           aria-valuemax={total}
           aria-valuetext={label}
-          tabindex="0"
+          tabindex="-1"
```

Comment:
Let's remove it completely following Josh's guidance (https://github.com/department-of-veterans-affairs/va.gov-team/issues/34872#issuecomment-1125432981):

> given it isn't interactive and that there are written steps, I don't see the need of it ever needing programmatic focus. Removing the `tabindex` completely should do the trick :smile: